fenzo-core/src/main/java/com/netflix/fenzo/ActiveVmGroups.java

```diff
@@ -52,13 +52,25 @@ private String getName() {
         activeVmGroupsMap.put(0, new ArrayList<VmGroup>());
     }
 
+    /**
+     * method to check if a VM Group lies in the VM Groups list searching via VM Group name
+     *
+     * @param vmg string containing the VM Group name to search
+     * @return VmGroup if found, null otherwise
+     */
     private VmGroup isIn(String vmg, List<VmGroup> list) {
         for(VmGroup g: list)
             if(g.getName().equals(vmg))
                 return g;
         return null;
     }
 
+    /**
+     * method to set all the VM Group names in the list vmGroups to active by creating new VmGroup object for the
```

Comment:
The description you provided, as it stands now, refers to internal implementation of how the groups list is maintained. The internal implementation (storing the list as the only entry in a map, keyed by "0") has no bearing on the behavior exposed to the caller. The behavior is fully described by the initial part of your description, "method to set all the VM Group names in the list vmGroups to active". Would you like to modify it to simplify the description? Also, make the first letter of the first word in the description upper case.